state/core_access.go

```diff
@@ -203,6 +195,15 @@ func (ca *CoreAccessor) SubmitPayForBlob(
 		return nil, fmt.Errorf("account for signer %s not found", author)
 	}
 
+	gas := cfg.GasLimit()
+	if gas == 0 {
+		msg, err := apptypes.NewMsgPayForBlobs(author.String(), appconsts.Version, libBlobs...)
```

Comment:
you don't need to pass app const version here, value is ignored anyway in function (❓ ) but passing this would just make it confusing.

Reply:
wtf lol